src/help.html

```diff
@@ -150,7 +150,18 @@ <h3>How to install my BeakBox</h3>
           green.
         </li>
       </ul>
-      <div class="youtube-player" data-id="_iG9LN9I2ug"></div>
+      <div>
+        <iframe
+          width="560"
+          height="315"
+          loading="lazy"
+          src="https://www.youtube.com/embed/_iG9LN9I2ug"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
```

Comment:
Unsure why any/most of these are needed and I would guess they give Google fingerprinting data for the end user which isn't ideal. Mainly the accelerometer, gyroscope, and clipboard-write values..

Reply:
These are standard parameters that google generates when you want to share YouTube embeds. Honestly, haven't looked at each of them individually and blindly trusted google that these attributes are there for the best performance